and-let.rkt

```diff
@@ -0,0 +1,13 @@
+#lang racket/base
+
+(provide and-let)
+
+(require syntax/parse/define "if-let.rkt" (for-syntax racket/base))
```

Comment:
Thanks for the PR!

I'm adding a `aand` form too, to keep the symmetry with other forms (`if-let` / `aif`, etc.), and I'll merge in 5 minutes.

Just out of curiosity, is there any specific reason for using `syntax/parse/define` instead of `syntax/parse`, since you are not using define-simple-macro?

Reply:
No, there isn’t. A previous version of this PR used `define-syntax-parser`, and I didn’t think to change the require back.

Reply:
Merged, thanks for the clarification :) !